services/state/roster/roster.proto

```diff
@@ -68,14 +79,20 @@ message RosterEntry {
     bytes gossip_ca_certificate = 3;
 
     /**
-     * The long term elliptic-curve cryptography
-     * public key for the node used in TSS.
+     * An ALT_BN128 elliptic curve public encryption key.<br/>
      * <p>
-     * This value SHALL be the DER encoding of the presented elliptic-curve
-     * public key that supports TSS<br/>
-     * This field is REQUIRED and MUST NOT be empty.
+     * The elliptic curve type may change in the future. For example, <br/>
+     * if the Ethereum ecosystem creates precompiles for BLS12_381,
+     * we may switch to that curve.
+     * <p>
+     * This value SHALL be specified according to EIP-196 and EIP-197 standards, <br/>
+     * See https://eips.ethereum.org/EIPS/eip-196#encoding and <br/>
+     * https://eips.ethereum.org/EIPS/eip-197#encoding
+     * <p>
```

Comment:
For _fields_ we should use line breaks between individual requirements, and a single paragraph break between the summary/description and the requirements.
```suggestion
     * https://eips.ethereum.org/EIPS/eip-197#encoding<br/>
```